client/src/screens/settings.screen.js

```diff
@@ -148,28 +160,22 @@ class Settings extends Component {
             <TouchableOpacity style={styles.imageContainer}>
               <Image
                 style={styles.userImage}
-                source={{ uri: 'https://facebook.github.io/react/img/logo_og.png' }}
+                source={{ uri: 'https://www.lehigh.edu/google/google_contacts.png' }}
```

Comment:
I'm going to use an icon for this instead - loading icon from random website probably not a great idea.